### PR TITLE
fix: adapt export and download tests to new hoops

### LINKED ISSUE #982: Fix import issues with HOOPS and DMS for new version
Resolve issues with https://github.com/ansys/pyansys-geometry/pull/981

### PR DESCRIPTION
**Pull request by b-matteo**
## Description
**Please provide a brief description of the changes made in this pull request.**

## Issue linked
Closes #982

## Checklist
- [x] I have tested my changes locally.
- [x] I have added necessary documentation or updated existing documentation.
- [x] I have followed the coding style guidelines of this project.
- [x] I have added appropriate unit tests.
- [x] I have reviewed my changes before submitting this pull request.
- [x] I have linked the issue or issues that are solved to the PR if any.
- [x] I have assigned this PR to myself.
- [x] I have added the minimum version decorator to any new backend method implemented.
- [x] I have made sure that the title of my PR follows [Conventional commits style](https://www.conventionalcommits.org/en/v1.0.0/#summary) (e.g. ``feat: extrude circle to cylinder``)

### CODE REVIEW

File: tests/integration/test_design_import.py [hunk 1]
```diff
@@ -144,7 +144,6 @@ def test_design_import_with_surfaces_issue834(modeler: Modeler):
         assert design.bodies[1].is_surface == True
 
 
-@pytest.mark.skip(reason="Get the OpenSSL GeometryService through before fixing hoops")
 def test_open_file(modeler: Modeler, tmp_path_factory: pytest.TempPathFactory):
     """Test creation of a component, saving it to a file, and loading it again to a
     second component and make sure they have the same properties."""
```
Comment (PipKat) [inline tests/integration/test_design_import.py:149]:
```suggestion
    """Test creation of a component.

    This method saves the open file to a new file. It, then loads the new
    file to a second component to make sure that they have the same
    properties."""
```
I hope I understood what was there. If not, I'm sure you understand the need for improving what was originally there.

**Review by PipKat — APPROVED** [review]

File: tests/integration/test_design_import.py [hunk 2]
```diff
@@ -204,17 +203,19 @@ def test_open_file(modeler: Modeler, tmp_path_factory: pytest.TempPathFactory):
...
         design.download(file, DesignFileFormat.STEP)
         design2 = modeler.open_file(file)
-        _checker_method(design, design2, False)
+        design3 = modeler.open_file("./tests/integration/files/import/twoCars.stp")
+        _checker_method(design2, design3, False)
 
         # IGES
-        #
+
         # TODO: Something has gone wrong with IGES
```
Comment (RobPasMue) [inline tests/integration/test_design_import.py:211]:
```suggestion
```

File: tests/integration/test_design_import.py [hunk 3]
```diff
@@ -204,17 +203,19 @@ def test_open_file(modeler: Modeler, tmp_path_factory: pytest.TempPathFactory):
...
         # TODO: Issue https://github.com/ansys/pyansys-geometry/issues/801
```
Comment (RobPasMue) [inline tests/integration/test_design_import.py:212]:
```suggestion
```

**Review by RobPasMue — DISMISSED** [review]

File: tests/integration/test_design_import.py [hunk 4]
```diff
@@ -200,21 +199,20 @@ def test_open_file(modeler: Modeler, tmp_path_factory: pytest.TempPathFactory):
...
-        design.download(file, DesignFileFormat.STEP)
-        design2 = modeler.open_file(file)
-        _checker_method(design, design2, False)
-
         # IGES
-        #
-        # TODO: Something has gone wrong with IGES
-        # TODO: Issue https://github.com/ansys/pyansys-geometry/issues/801
-        #
+
```
Comment (RobPasMue) [inline tests/integration/test_design_import.py:203]:
```suggestion
        #
        # TODO: Something has gone wrong with IGES
        # TODO: Issue https://github.com/ansys/pyansys-geometry/issues/801
```

**Review by RobPasMue — DISMISSED** [review]

**Review by RobPasMue — APPROVED** [review]
LGTM! Merging

**Pull request merged**